app/models/user.rb

```diff
@@ -114,6 +114,10 @@ def self.reset_badge_counters(user)
     User.reset_counters user.id, :badge_acquisitions
   end
 
+  def uminho_student(course)
```

Comment:
@frmendes Is this it? I was trying to wrap my head around if the argument was really needed...